### PR TITLE
LKE-3737 add config to integrate CSV plugin to the CI

### PR DESCRIPTION
**Pull request by aminerhanemi**

### CODE REVIEW

**Comment by aminerhanemi** [comment]
running npm build locally fail but we don't any error. @enys  can you check if the script are correct ?

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  // General
+  projectName = "linkurious/lke-plugin-csv-importer"
+  runUnitTests = false
+  runE2eTests = false
+
+  createGitTag = true
+
+  //static asset upload
+  runPrivateNpmPublish = false
+  binaries = ["lke-plugin-data-table.lke"]
```
Comment (francescoinfante) [inline Jenkinsfile:13]: @aminerhanemi you need to remove the reference to the data-table here

Comment (francescoinfante) [inline Jenkinsfile:13]: @aminerhanemi it should be fixed to the csv import

File: package.json [hunk 2]
```diff
@@ -2,16 +2,22 @@
...
     "clean": "rm -rf csv-importer.lke",
-    "build": "npm-run-all clean build:public build:backend copyManifest",
+    "build:project": "npm-run-all clean build:public build:backend copyManifest",
     "build:public": "tsc -b src/public && npm run copy-assets && cd src/public/dist && browserify *.js --debug -o ../../../csv-importer.lke/public/index.js",
     "copy-assets": "cp -R src/public/assets csv-importer.lke/public && cp src/public/index.html csv-importer.lke/public",
     "build:backend": "tsc -b src/backend && npm run installProduction",
     "installProduction": "cp package.json csv-importer.lke && cd csv-importer.lke",
     "copyManifest": "cp -R src/manifest.json csv-importer.lke/",
+    "lint:ci-deactivated": "eslint -f checkstyle -o reports/checkstyle.xml  .",
+    "build": "npm run build:project && npm pack && gunzip -qf -c linkurious-lke-csv-importer-$(cat .version).tgz > lke-plugin-csv-importer.lke",
```
Comment (francescoinfante) [inline package.json:16]: linkurious-lke-csv-importer- is not the name of the package, it should be: lke-plugin-csv-importer-

**Review by francescoinfante — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (enys) [reply Jenkinsfile:13]: and basicaly match the 'projectName'

File: package.json (hunk 2 above)
Comment (enys) [reply package.json:16]: Please don't run clean on build.

File: package.json [hunk 3]
```diff
@@ -2,16 +2,22 @@
   "name": "lke-plugin-csv-importer",
```
Comment (enys) [inline package.json:2]: Package must be scoped.

**Review by enys — CHANGES_REQUESTED** [review]
npm Package scoping is mandatory

**Review by enys — APPROVED** [review]
Lets just look at the state of the Lint, but LGTM otherwise.

**Comment by aminerhanemi** [comment]
for the linter I will create a separate task

**Pull request merged**